content/blog/digib0y/2018-02-02_gci_2017_wrap_up_report.md

```diff
@@ -0,0 +1,65 @@
++++
+type = "blog"
+title = "GCI 2017 Wrap Up Report"
+author = "digib0y"
+date = "2018-02-02 00:13:41+05:30"
+tags = ["haiku", "software","GCI","Google","students","Code-In","Google Code-In","GCI2016"]
++++
+
+
+<p>Like all good things Google Code-In 2017 comes to an end, this year the program attracted over 3529 students in total for Google Code-In and out of those about ⅓ of the students claimed at least one task for Haiku, with 255 students completing at least one Haiku task. Haiku is also proud of its history of getting selected each year in <a href="http://codein.withgoogle.com">Google’s Code-In</a>.
+
+<p>Just like the other 24 organizations, Haiku picked winners out of the 10 students who completed the most tasks for Haiku, finalists and grand prize winners were picked collectively by the diligent team of Haiku’s 20+ mentors. The grand prize winners will be flown to Northern California to visit Google’s headquarters, enjoy a day of adventure in San Francisco, and meet one or two of their mentors and Google engineers.</p>
+
+<h3>Haiku’s Grand Prize Winners</h3>
+<ul>
+<li>
+Owen Pan from United States</li>
+<li>Hiếu Đức Lê from Vietnam</li>
+</ul>
+
+<h3>Haiku’s Finalists</h3>
```

Comment:
I would combine these two sections. "Rest of Haiku's top ten finishers" is a little confusing.

Reply:
Which two sections are you talkin g about?

I am planning to go with these four sections:

Haiku’s Grand Prize Winners
Haiku's Runner UP
Haiku’s Finalists
Rest of Haiku’s top ten finishers